src/components/picture/_children/source.test.jsx

```diff
@@ -0,0 +1,33 @@
+import { render } from "@testing-library/react";
+
+import Source from "./source";
+
+describe("Source subcomponent", () => {
+	it("render unresized image an auth key has not been provided", () => {
+		const { container } = render(
+			<Source src="test.jpeg" media="(max-width: 799px)" resizedOptions={{}} />
```

Comment:
```suggestion
			<Source src="test.jpeg" media="(max-width: 799px)" />
```

Reply:
@matthewroach : Thank you! There were three tests using these empty objects. I removed these.